tasks/registry/index.ts

```diff
@@ -0,0 +1,40 @@
+import { task, types } from "hardhat/config";
+
+export type RegistryAddParams = {
+  address: string;
+  value?: string | number | bigint;
+};
+
+task("registry:add", "adds the specified hyperdrive instance to the registry")
+  .addParam(
+    "address",
+    "address of the hyperdrive instance to add",
+    undefined,
+    types.string,
+  )
+  .addOptionalParam(
```

Comment:
Might be better for this to not be optional. I think we want people to think about this whenever they run the script